src/components/pages/Login.js

```diff
@@ -1,11 +1,75 @@
-import React from 'react';
+import React from "react";
+import { useState } from "react";
+import { Redirect } from "react-router-dom";
+import axios from "axios";
+
+function Login(props) {
+  const [credentials, changeCredentials] = useState({
+    email: "",
+    password: ""
+  });
+
+  const [status, changeStatus] = useState({
+    loading: false,
+    error: false
+  });
+
+  const submitLogin = event => {
+    event.preventDefault();
+    changeStatus({ ...status, loading: true });
+    axios
```

Comment:
We'll need to refactor this with the client that will take care of the login and token validation. @AnthonyJCampbell will you take it from here? 🔥

Reply:
yes, whenever anthony is finished we will include it

Reply:
For sure, there are separate request handlers that you can import and simply call with the requisite params. Easy-Peasy 🔥